func_smooth.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+def smooth(x, window_len=11, window='hanning'):
+    '''
+    Using inbuilt convolve function of numpy
+    input:
+        x: signal
+        window_len: the dimension of the smoothing window; should be an odd integer
+        window: type of window from 'flat', 'hanning', 'hamming', 'bartlett', 'blackman'
+    output:
+        the smoothed signal
+    '''
+
+    if window_len < 3:  return x
+
+    if x.ndim != 1: raise (StandardError('smooth only accepts 1 dimension arrays.'))
+    if x.size < window_len:  raise (StandardError('Input vector smaller than window size.'))
+    win_type = ['flat', 'hanning', 'hamming']
+    if window not in win_type: raise( StandardError('Unknown window type'))
```

Comment:
Nice to see that you are raising standard errors.

Also, add available options along with 'unknown window type' in this comment.